lib/src/core/toastification_manager.dart

```diff
@@ -20,6 +20,9 @@ class ToastificationManager {
 
   OverlayEntry? _overlayEntry;
 
+  /// store the item ids that are waiting for overlay
+  final List<String> _delayedIds = [];
```

Comment:
Do we want to specify what we are saving them for? (To avoid showing toasts that have already been dismissed)